src/rgw/rgw_rest_s3.cc

```diff
@@ -586,15 +588,11 @@ void RGWPutObjTags_ObjStore_S3::send_response()
 
 void RGWDeleteObjTags_ObjStore_S3::send_response()
 {
-  int r = op_ret;
-  if (r == -ENOENT)
-    r = 0;
-  if (!r)
-    r = STATUS_NO_CONTENT;
-
-  set_req_state_err(s, r);
+  if (op_ret)
+    set_req_state_err(s, op_ret);
   dump_errno(s);
-  end_header(s, this);
+  end_header(s, this, "application/xml");
```

Comment:
@selvakumaar5496 can you elaborate how these two line changes is helping?

Reply:
Followed a common template that is being used across all tagging send_response() function